pages/create-list.tsx

```diff
@@ -95,16 +98,17 @@ export const CreateListPage: NextPage = () => {
           />
         </HStack>
       </Stack>
-      {visibleServices.map((service) => (
-        <ServiceItem
-          key={service.id}
-          service={service}
-          onAlertOpen={onAlertOpen}
-          setSelectedService={setSelectedService}
-        />
-      ))}
+      <Grid templateColumns="repeat(3, 1fr)" gap="32px">
+        {visibleServices.map((service) => (
+          <ResourceCard
+            key={service.id}
+            resource={service}
+            saveResource={() => saveResource(service)}
+          />
+        ))}
+      </Grid>
```

Comment:
Maintains functionality for opening the "Create a list" drawer with the `ResourceCard` instead of `ServiceItem`